test/parallel/test-path-basename.js

```diff
@@ -0,0 +1,91 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
I believe new files don't need this.

Reply:
Even when the content is entirely copied from a file with the licence header? I would be more than happy to remove those.

Reply:
Yep, the way we've been handling it is keeping it only in the preexisting files